medusa/server/core.py

```diff
@@ -36,6 +40,9 @@ def get_apiv2_handlers(base):
         # /api/v2/series/tvdb1234/episode
         EpisodeHandler.create_app_handler(base),
 
+        # /api/v2/series/tvdb1234/season
+        SeasonHandler.create_app_handler(base),
```

Comment:
This should have been added separately, @ratoaq2 any reason why this wasn't added before now? Didn't we support seasons before now?